libs/language-server/src/stdlib/builtin-constrainttypes/RegexConstraint.jv

```diff
@@ -0,0 +1,15 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+/**
+* Limits the values complying with a regex. Only values that comply with the regex are considered valid.
+*
+* @example Text that complies with the IFOPT (Identification of Fixed Objects in Public Transport) DIN EN 28701:2012 format.
```

Comment:
I think we were told in MADe that the format is wrong for IFOPT actually? Maybe can just use a different example like an IPv4 address.